src/services/cartService.js

```diff
@@ -0,0 +1,11 @@
+const database = require("typeorm");
+const cartDao = require("../models/cartDao");
+
+const cart = async (userId, productId, quantity) => {
```

Comment:
함수는 "cart 목록을 불러와라.", "cart를 생성하라" 등 Action의 의미를 담고 있습니다. 그렇기 때문에 이름을 명사가 아니라 동사로 만들어야 합니다. 또한 명확하게 cart 한 개를 가져오는 것인지 여러개를 가져오는 것인지도 단수와 복수 그대로 표현해주는 것이 좋습니다.

```suggestion
const getCarts = async (userId, productId, quantity) => {
```